### ISSUE TITLE
Automule question

### ISSUE THREAD

**Issue by doktormatte** [kolton/d2bot-with-kolbot#1096]
So I have 6 key pairs and running 6 mf bots. Now I want to use automule. Do I need an extra key for that or does the automule process work around that, ie bot makes game, stays in, drops items, logs out, mule logs in, comes into game and picks items up?

**Comment by gbgodoy** [comment]
yes, you need an extra key set available to open the mule profile when needed because the key gonna be in use.

**Comment by braxIRL** [comment]
thank you for the response!

**Comment by texxxmex** [comment]
You don’t necessarily need another key. It can also be set up to turn off another bit and use that key for the mule. AFAIK it doesn’t perm a game to mule, though it’s been a bit

**Comment by doktormatte** [comment]
Well guess we could find out via trial and error...

**Comment by doktormatte** [comment]
So a possible solution if I want to enable automule for all my bots without having to obtain an extra key would be to use one key for three bots (one of the other three keys) and another one for the other three bots (one of the three keys in use by the first three bots) ? How exactly are the items exchanged? Do two bots enter a game at once, one of them drops and the other one picks it up immediately?

**Comment by romaingallay** [comment]
Yes to both of your questions.
